diff --git a/Services/Hardware/BrightnessService.qml b/Services/Hardware/BrightnessService.qml
index 44e40f3..2ea5e96 100644
--- a/Services/Hardware/BrightnessService.qml
+++ b/Services/Hardware/BrightnessService.qml
@@ -478,7 +478,7 @@ Singleton {
     }
 
     function setBrightness(value: real): void {
-      value = Math.min(1, value);
+      value = Math.max(0 , Math.min(1, value));
       var rounded = Math.round(value * 100);
 
       // Always update internal value and trigger UI feedback immediately
