diff --color -ur orig/constants.js new/constants.js
--- a/main/background-work/common/application/constants.js	2025-07-20 09:56:22.931956731 +0300
+++ b/main/background-work/common/application/constants.js	2025-07-20 09:55:41.528196868 +0300
@@ -12,6 +12,9 @@
         'folder.png',
         'folder.jpg',
         'folder.jpeg',
+        'albumart.png',
+        'albumart.jpg',
+        'albumart.jpeg',
         '%filename%.png',
         '%filename%.jpg',
         '%filename%.jpeg',
diff --color -ur orig/constants.ts new/constants.ts
--- a/src/app/common/application/constants.ts	2025-07-20 09:56:30.959123545 +0300
+++ b/src/app/common/application/constants.ts	2025-07-20 09:55:49.337576781 +0300
@@ -63,6 +63,9 @@
         'folder.png',
         'folder.jpg',
         'folder.jpeg',
+        'albumart.png',
+        'albumart.jpg',
+        'albumart.jpeg',
         '%filename%.png',
         '%filename%.jpg',
         '%filename%.jpeg',
