--- a/src/neroprefixsettings.cpp	2025-07-20 09:20:53.457015962 +0300
+++ b/src/neroprefixsettings.cpp	2025-07-20 09:21:57.386760437 +0300
@@ -95,26 +95,6 @@
             winVersionListBackwards.append(ui->winVerBox->itemText(i-1));
     }
 
-    // FSR scalers are only implem'd in GE-Proton
-    if(!ui->prefixRunner->currentText().startsWith("GE-Proton")) {
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRperformance, false);
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRbalanced, false);
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRquality, false);
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRhighquality, false);
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRhigherquality, false);
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRhighestquality, false);
-        SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingFSRcustom, false);
-        // HACK: offset by one when this is a shortcut settings panel and not prefix settings
-        if(ui->setScalingBox->findText("Integer Scaling") == 1) SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingIntegerScale, false);
-        else SetComboBoxItemEnabled(ui->setScalingBox, NeroConstant::ScalingGamescopeWindowed, false);
-    }
-
-    // Wayland requires base version to be Proton 10+
-    if(!ui->prefixRunner->currentText().startsWith("GE-Proton10") && !ui->prefixRunner->currentText().startsWith("Proton 10")) {
-        ui->toggleWayland->setEnabled(false);
-        ui->toggleWaylandHDR->setEnabled(false);
-    }
-
     resValidator = new QIntValidator(0, 32767);
     ui->fsrCustomH->setValidator(resValidator);
     ui->fsrCustomW->setValidator(resValidator);
